pkg/mkpersist/Dockerfile.in

```diff
@@ -0,0 +1,9 @@
+FROM alpine:3.8
```

Comment:
We no longer need it to be *.in

Reply:
Also, I'd rather switch to the latest Alpine 3.10

Reply:
Will change to alpine:3.10.3 as was done for rsyslog container.